.circleci/config.yml

```
@@ -30,21 +30,19 @@ jobs:
       - run: DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain BUILD_FROM_INTERMEDIATE=y make build build_serialize
       - restore_cache:
           keys:
-            - v1.3-inputdata-{{ checksum "download_inputdata.sh" }}-{{ checksum "requirements.txt"}}
+            - v1.4--{{ checksum "requirements.txt"}}
       - run: 
           name: "Install Python test dependencies"
           command: |
             pyenv versions
             [ -d /opt/circleci/.pyenv/versions/3.6.2 ] || pyenv install 3.6.2
             pyenv global 3.6.2
             pip install -r requirements.txt
-      - run: bash download_inputdata.sh
       - save_cache:
           paths:
-            - ./fv3gfs-data-docker_2020-01-13.tar.gz
             - $FV3CONFIG_CACHE_DIR
             - /opt/circleci/.pyenv/versions/3.6.2
-          key: v1.3-inputdata-{{ checksum "download_inputdata.sh" }}-{{ checksum "requirements.txt"}}
+          key: v1.3-{{ checksum "requirements.txt"}}
```

To match the above key.
```suggestion
          key: v1.4-{{ checksum "requirements.txt"}}
```